30-remote-site-setup.sh

```diff
@@ -98,7 +98,7 @@ for ruser in $users; do
     setup_ssh_config
     [[ $cvmfs_wn_client -eq 0 ]] || setup_endpoints_ini
     # $REMOTE_BATCH needs to be specified in the environment
-    bosco_cluster "${override_opts[@]}" -a "${ruser}@$REMOTE_HOST" "$REMOTE_BATCH"
+    bosco_cluster "${override_opts[@]}" -b "$REMOTE_BOSCO_DIR" -a "${ruser}@$REMOTE_HOST" "$REMOTE_BATCH"
```

Comment:
Is `REMOTE_BOSCO_DIR` guaranteed to be provided in the environment?

That is, should this `-b` option be omitted if `$REMOTE_BOSCO_DIR` is empty or unset?

Reply:
It'll always be specified by SLATE, which is currently the only use case, but it doesn't hurt to make this more robust. Addressed in the latest commit